stingray/crosscorrelation.py

```diff
@@ -206,7 +210,7 @@ def _make_corr(self, lc1, lc2):
             variance2 = np.var(lc2.counts) - np.mean(lc2.counts_err)**2
             self.corr = self.corr / np.sqrt(variance1 * variance2) / lc1_counts.size
 
-    def cal_timeshift(self, dt=1.0):
+    def cal_timeshift(self, dt: float = 1.0) -> Tuple[float, np.ndarray]:
```

Comment:
There are three return values here

Reply:
I will add it, but it should be added to the documentation under this function as the documentation does not mention the third return value called "int n" which is the length as I understood from the code.